### DISCUSSION TITLE
Is it possible to clamp only one value during spring animation?

### DISCUSSION THREAD

**Discussion by xzilja** [nandorojo/moti#311] [Q&A] [answered]
Hey 👋 

I've got following animation that essentially "scale fades" element in with a spring transition. However, while element is "bouncing" changes in opacity also occur. I was wondering if it is possible to "clamp" value of opacity somehow that once it fully animates in (i.e. value is 1) it never animates below that value anymore.

```tsx
<MotiView
  from={{ scale: 0, opacity: 0 }}
  animate={{ scale: 1, opacity: 1 }}
  transition={ type: 'spring', delay: 500 }
/>
```

**Comment by nandorojo** [comment, marked as answer]
![IMG_4911](https://github.com/nandorojo/moti/assets/13172299/11cd8a4e-27a6-4314-95c1-d90741b9c909)

**Reply by xzilja** [reply]
Ah, this is perfect 👍 thank you
